charts/super-agent/README.md

```diff
@@ -0,0 +1,81 @@
+[![Community Plus header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Plus.png)](https://opensource.newrelic.com/oss-category/#community-plus)
+
+# super-agent
+
+![Version: 0.0.0-beta](https://img.shields.io/badge/Version-0.0.0--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+
+Bootstraps New Relic' Super Agent
+
+# Helm installation
+
+You can install this chart using directly this Helm repository:
+
+```shell
+helm repo add newrelic https://newrelic.github.io/helm-charts
+helm upgrade --install newrelic/CHART-TEMPLATE -f your-custom-values.yaml
+```
+
+## Values managed globally
+
+This chart implements the [New Relic's common Helm library](https://github.com/newrelic/helm-charts/tree/master/library/common-library) which
+means that it honors a wide range of defaults and globals common to most New Relic Helm charts.
+
+Options that can be defined globally include `affinity`, `nodeSelector`, `tolerations`, `proxy` and others. The full list can be found at
+[user's guide of the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md).
```

Comment:
isn't this a bit misleading? Since we are including Flux how are we making sure the features are actually implemented?

Reply:
I rephrased it on [#d2e49c1](https://github.com/newrelic/helm-charts/pull/1175/commits/d2e49c17d289c548346804a27429633fbe279b38)

Let me know what you think.

Reply:
I like the disclaimer regarding flux! From my point of view it is enough 😄